multiversx_sdk/core/config.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class LibraryConfig:
+    """
+    Global configuration of the library.
+
+    Generally speaking, this configuration should only be altered on exotic use cases
+    it can be seen as a collection of constants (or , to be more precise, rarely changed variables) that are used throughout the library.
+
+    Never alter the configuration within a library!
```

Comment:
Indeed. Only in an _end_ application.

Altering within a library (several times) might lead to ambiguity and conflicts.